### ISSUE TITLE
Docs :: GraphQLResult is not defined

### ISSUE THREAD

**Issue by sgpinkus** [graphql/graphql.github.io#1049]
### Description

The [API doc](https://graphql.org/graphql-js/graphql/#graphql) for the graphql() function says it returns `Promise<GraphQLResult>`. But GraphQLResult is not defined, and does not exist in the code base AFAICT.

### Motivation

Completeness, accuracy.

**Comment by saihaj** [comment]
Good catch. `ExecutionResult` is what is returned. Docs for [`graphql-js`](https://github.com/graphql/graphql-js) are outdated on this site as well as in [`graphql-js/docs`](https://github.com/graphql/graphql-js/tree/main/docs). We are currently migrating to TypeScript. Generating docs (using ts docs or other tools) and providing more examples is next task in [`v16` release plan](https://github.com/graphql/graphql-js/issues/2860)

**Comment by sgpinkus** [comment]
> We are currently migrating to TypeScript.

OK cool. Since then I've noticed a bunch of other inconsistencies in those docs. Like for example the [docs say GraphQlSchemaConfig](https://github.com/graphql/graphql-js/blob/main/docs/APIReference-TypeSystem.md#graphqlschema) looks like this:

```
type GraphQLSchemaConfig = {
  query: GraphQLObjectType;
  mutation?: ?GraphQLObjectType;
}
```

when it's actually this on master:

```
export type GraphQLSchemaConfig = {|
  description?: ?string,
  query?: ?GraphQLObjectType,
  mutation?: ?GraphQLObjectType,
  subscription?: ?GraphQLObjectType,
  types?: ?Array<GraphQLNamedType>,
  directives?: ?Array<GraphQLDirective>,
  extensions?: ?ReadOnlyObjMapLike<mixed>,
  astNode?: ?SchemaDefinitionNode,
  extensionASTNodes?: ?$ReadOnlyArray<SchemaExtensionNode>,
  ...GraphQLSchemaValidationOptions,
|};
```

Since it's a wip I won't bother making anymore issues for docs. Feel free to close this if you want.

Cheers.

**Comment by yaacovCR** [comment]
Closing this issue, graphql-js website now tracked at the graphql-js repo and docs are now generated directly from source.
